scripts/publish_video/helper.js

```diff
@@ -59,6 +59,68 @@ function hasPostingAuthority(account) {
   });
 }
 
+async function getHiveContentBeneficiaries(author, permlink) {
+  return new Promise((resolve, reject) => {
+    hive.api.getContent(author, permlink, function (err, result) {
```

Comment:
preferably use dhive over hivejs

Reply:
Is this a blocker? I've already tested with `hive.api` and it is working as expected.
with dhive, I may have to rewrite function.

Reply:
Not a blocker, but going forward dhive should be used over hivejs for most use cases.

Reply:
Understood. We're anyway switching to new frontn-end & back-end. As a result, we'll have dhive everywhere.